ci-operator/step-registry/upi/deprovision/azure/upi-deprovision-azure-commands.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set +e
+echo "Deprovisioning cluster ..."
+ls ${ARTIFACT_DIR}/metadata.json
+if [[ -f ${ARTIFACT_DIR}/metadata.json ]]; then
+    INFRA_ID="$(jq -r .infraID ${ARTIFACT_DIR}/metadata.json)"
```

Comment:
I think you want `${SHARED_DIR} instead of `${ARTIFACT_DIR}`.
```suggestion
ls ${SHARED_DIR}/metadata.json
if [[ -f ${SHARED_DIR}/metadata.json ]]; then
    INFRA_ID="$(jq -r .infraID ${SHARED_DIR}/metadata.json)"
```

Reply:
Ah damn silly mistake. Thanks!